libraries/aws_s3_buckets.rb

```diff
@@ -0,0 +1,108 @@
+require '_aws'
+
+class AwsS3Buckets < Inspec.resource(1)
+  name 'aws_s3_buckets'
+  desc 'Verifies settings for AWS S3 Buckets in bulk'
+  example "
+    describe aws_s3_buckets do
+      its('buckets.all') { should be_in ['logging_bucket', 'another_bucket'] }
+      it { should_not have_public_buckets }
+    end
+  "
+  include AwsResourceMixin
```

Comment:
We don't usually use AwsResourceMixin for plural resources; there is less commonality, and the constructor is a bit wonky for this use case.  

I think you've got a lot working here, doing some awkward things to handle AwsResourceMixin's influence.  We can detangle it, or not.